arch/arm64/configs/fancom_whc_defconfig

```diff
@@ -972,137 +720,15 @@ CONFIG_USB_HIDDEV=y
 CONFIG_I2C_HID=m
 CONFIG_USB=y
 CONFIG_USB_ANNOUNCE_NEW_DEVICES=y
-CONFIG_USB_MON=m
```

Comment:
Deze moet aan blijven voor usb monitoring van de WHC. Of zat dat op de HMI?

Reply:
Added again